README.md

```diff
@@ -122,6 +123,54 @@ From the application directory run:
 bin/console fom:user:resetroot
 ```
 
+## Docker
+
+Run this command to start a local mapbender instance in a docker container using an internal sqlite database.
+
+```bash
+docker run mapbender/mapbender
+```
+
+### external database 
+
+To run a Mapbender docker container configured to use an external database we will use the MAPBENDER_DATABASE_URL environment variable.
+
+1.  Create the docker network
+    `docker network create -d bridge mapbender`
+
+2.  Start the database
+    `docker run --name db --network mapbender -p 55432:5432 -e POSTGRES_USER=pguser -e POSTGRES_PASSWORD=pgpass -e POSTGRES_DB=postgres postgis/postgis:14-3.4`
+
+3.  Start mapbender
+    `docker run --name mapbender --network mapbender -p 80:8080 -e MAPBENDER_DATABASE_URL="pgsql://pguser:pgpass@db:5432/postgres" mapbender/mapbender`
+
+4.  Create Mapbender database tables
+    `docker exec mapbender php application/bin/console doctrine:schema:update --force`
+
+5.  Initialize Mapbender database
+    `docker exec mapbender php application/bin/console mapbender:database:init -v`
+
+### docker/docker-compose.pgsql.yml
+
+If you have connected the mapbender container to an external database that does not have the required tables yet, like the setup started by the following command:
```

Comment:
Please check the sentence. 

, like the setup started by the following command:

sound a bit strange.

Reply:
It did make sense when reading the next step, but was probably bad style. Changed it